pkgs/games/tennix/default.nix

```diff
@@ -1,20 +1,21 @@
-{lib, stdenv, fetchurl, SDL, SDL_mixer, SDL_image, SDL_ttf, SDL_net, python2 } :
+{ lib, stdenv, fetchgit, which, SDL, SDL_mixer, SDL_image, SDL_ttf, SDL_net, python3 } :
 
 stdenv.mkDerivation rec {
   pname = "tennix";
-  version = "1.1";
-  src = fetchurl {
-    url = "https://icculus.org/tennix/downloads/tennix-${version}.tar.gz";
-    sha256 = "0np5kw1y7i0z0dsqx4r2nvmq86qj8hv3mmgavm3hxraqnds5z8cm";
+  version = "1.3.1";
+
+  src = fetchgit {
+    url = git://repo.or.cz/tennix.git;
+    rev = "refs/tags/tennix-${version}";
+    sha256 = "sha256-U5+S1jEeg+7gdM1++dln6ePTqxZu2Zt0oUrH3DIlkgk=";
   };
 
-  buildInputs = [ python2 SDL SDL_mixer SDL_image SDL_ttf SDL_net ];
+  nativeBuildInputs = [ which ];
 
-  patches = [ ./fix_FTBFS.patch ];
+  buildInputs = [ python3 SDL SDL_mixer SDL_image SDL_ttf SDL_net ];
 
-  preConfigure = ''
-    makeFlags="PREFIX=$out"
-    installFlags="PREFIX=$out install"
+  configurePhase = ''
+    ./configure --prefix $out
   '';
```

Comment:
Please use configureFlags instead

Reply:
The configure script cannot parse `--prefix=/nix/store/…`. As far as I know I can only append with configureFlags and `--prefix=/nix/store…` is always included.